apps/voting/app/src/App.js

```diff
@@ -146,45 +139,37 @@ class App extends React.Component {
   render() {
     const {
       app,
-      pctBase,
-      supportRequiredPct,
-      userAccount,
-      votes,
-      voteTime,
+      settingsLoaded,
       tokenDecimals,
       tokenSymbol,
+      userAccount,
+      votes,
     } = this.props
 
     const {
       createVoteVisible,
       currentVoteId,
-      settingsLoaded,
       tokenContract,
       voteSidebarOpened,
       voteVisible,
       userAccountVotes,
     } = this.state
 
-    const displayVotes = settingsLoaded && votes.length > 0
-    const supportRequired = settingsLoaded ? supportRequiredPct / pctBase : -1
+    const now = new Date()
```

Comment:
Something to solve later: I was getting spooked we weren't re-rendering enough, to check for cases where a vote goes from open to closed when the app is open.